src/themes/default/components/Carousel/_carousel.scss

```diff
@@ -19,8 +19,9 @@
     }
 
     .Text {
-        border-radius: 0;
+        border-radius: 10px;
         width: 100%;
+        height: 110px;
```

Comment:
hm why do you need height? carousels grow based on there contents if i recall?

Reply:
Belinda wants a fixed height on these, but let me double check on that.